ctapipe/reco/FitGammaHillas.py

```diff
@@ -337,27 +336,78 @@ def fit_origin_minimise(self, seed=(0, 0, 1), test_function=neg_angle_sum):
 
         return np.array(linalg.normalise(self.fit_result_origin.x)) * u.dimless
 
-    def fit_core(self, seed=(0, 0), test_function=dist_to_traces):
+    def fit_core_crosses(self, unit=u.m):
+        ''' calculates the core position as the least linear square solution of an
+        (over-constrained) equation system
+
+        Algorithm:
```

Comment:
this may cause a sphinx error (it seems to dislike any non-standard section names in a docstring).  I think the allowed ones are found here: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

I think renaming this "Notes" (with no :) should be fine